function-app-container/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM microsoft/azure-cli:2.0.47
+
+LABEL version="1.0.0"
+LABEL maintainer="Vincent Arockiadass"
```

Comment:
Should we say Microsoft? This might be relevant for all actions. Can you check with Pulkit on this.

Reply:
Changed to "Microsoft Corporation"